resources/views/about/terms-of-service.blade.php

```diff
@@ -9,8 +9,459 @@
+                ),
+            ) !!}</li>
+            <li>{!! Str::inlineMarkdown(
+                __(
+                    '**Severability.** If any provision of these Terms is found by a court of competent jurisdiction to be invalid, the parties nevertheless agree that the court should endeavor to give effect to the parties’ intentions as reflected in the provision, and the other provisions of these Terms remain in full force and effect.',
+                ),
+            ) !!}</li>
+            <li>{!! Str::inlineMarkdown(
+                __(
+                    '**Limitation Period.** You agree that regardless of any statute or law to the contrary, any claim or cause of action arising out of or related to use of the Platform or these Terms must be filed within one (1) year after such claim or cause of action arose or be forever barred.  A printed version of this agreement and of any notice given in electronic form will be admissible in judicial or administrative proceedings based upon or relating to this agreement to the same extent and subject to the same conditions as other business documents and records originally generated and maintained in printed form.',
+                ),
+            ) !!}</li>
+            <li>{!! Str::inlineMarkdown(
+                __(
+                    '**Assignment.** You may not assign these Terms without the prior written consent of IRIS, but IRIS may assign or transfer these Terms, in whole or in part, without restriction.',
+                ),
+            ) !!}</li>
+            <li>{!! Str::inlineMarkdown(
+                __(
+                    '**Interpretation.** The section titles in these Terms are for convenience only and have no legal or contractual effect.',
+                ),
+            ) !!}</li>
+            <li>{!! Str::inlineMarkdown(
+                __(
+                    '**Waiver.** No failure or delay by IRIS in exercising any right hereunder will waive any further exercise of that right.',
+                ),
+            ) !!}</li>
+            <li>{!! Str::inlineMarkdown(
+                __(
+                    '**Notices.** All notices or approvals required or permitted under these Terms will be in writing and delivered by email transmission, and in each instance will be deemed given upon receipt. All notices or approvals will be sent to IRIS at **[[:email](:email)]**. Notices to you may be made via either email or regular mail. The Platform may also provide notices to you of changes to these Terms or other matters by displaying notices or links to notices generally on the Platform.',
```

Comment:
Clicking on the email link directs to "/about/support@accessibilityexchange.ca" which is a page doesn't exist, and shows 404. I think we should open an email form when the link is clicked.